.github/workflows/ci_docs.yml

```diff
@@ -0,0 +1,45 @@
+name: Nim Docs CI
+on:
+  pull_request:
+    # Run only on changes on these files
+    paths:
+      - 'lib/*.nim'
```

Comment:
@narimiran should that' be lib/**/*.nim?
https://help.github.com/en/actions/reference/workflow-syntax-for-github-actions#filter-pattern-cheat-sheet
ditto below

Reply:
Already fixed in https://github.com/nim-lang/Nim/pull/13809